src/Model/Car.js

```diff
@@ -0,0 +1,25 @@
+export default class Car {
+  #name;
+
+  #moveCount = 0;
+
+  constructor(name) {
+    if (name.length > 5) {
+      throw new Error('[ERROR] 자동차 이름은 5자 이하여야 합니다.');
+    }
+
+    this.#name = name;
+  }
+
+  getName() {
+    return this.#name;
+  }
```

Comment:
name과 moveCount는 인스턴스마다 다른 값을 가질 수 있기 때문에, Car 클래스의 private 필드로 정의하는 것보다 인스턴스 변수로 정의하는 게 어떨까 싶어요!

constructor에서 정의해주면 인스턴스의 변수로 사용할 수 있어요.

```suggestion
  constructor(name) {
    if (name.length > 5) {
      throw new Error('[ERROR] 자동차 이름은 5자 이하여야 합니다.');
    }

    this.name = name;
    this.moveCount = 0;
  }

  getName() {
    return this.name;
  }
```

Reply:
`static` 키워드를 붙이지 않은 클래스 필드는 모두 인스턴스 필드로 추가되는 것으로 알고 있습니다!

`name`, `moveCount` 필드 모두 인스턴스 필드가 되는 것이 맞습니다!

Reply:
아 그렇군요! 몰랐는데 감사합니다.

혹시 그럼 constructor 안에 name만 있는데, moveCount는 안 넣으신 이유가 궁금합니다!

Reply:
`name`의 경우 외부에서 값을 주입받아 초기화를 해주어야해서 constructor에서 초기화를 시켜주었고 `moveCount`는 항상 0으로 초기화되는 필드라 constructor가 아닌 클래스 필드에서 직접 0으로 초기화해주었습니다!

Reply:
그렇군요! 알려주어 감사합니다.ㅎㅎ